src/client/dfuse/pil4dfs/hook_int.h

```diff
@@ -5,6 +5,8 @@
  * SPDX-License-Identifier: BSD-2-Clause-Patent
  */
 
+
+/* dummy change */
```

Comment:
Notice if you view this file in the `Files changed` tab, the annotation for the copyright is inline

Reply:
Looks good.